hubblestack/daemon.py

```diff
@@ -33,12 +33,14 @@
 import hubblestack.log
 import hubblestack.hec.opt
 import hubblestack.utils.stdrec
+import hubblestack.utils.osquery_lib
 from hubblestack import __version__
 from croniter import croniter
 from datetime import datetime
 from hubblestack.hangtime import hangtime_wrapper
 import hubblestack.status
 import hubblestack.saltoverrides
+import hubblestack.hubble_constants
```

Comment:
Why do we need this import?

Reply:
This is required so that this is available to nebula_osquery as an import.